packages/masks/src/typings/styled-components.d.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'styled-components';
+
+declare module 'styled-components' {
```

Comment:
Should this live in the type package?

Reply:
Eventually this will have to live in the `design-system` package (once we start migrating it), which is where our theme is defined.

The `types` package itself will likely go away at some point, with types being defined in their respective packages again (e.g. pattern types will be in patterns package)